lib/matplotlib/tests/test_gridspec.py

```diff
@@ -9,6 +10,16 @@ def test_equal():
     assert gs[:, 0] == gs[:, 0]
 
 
+def test_update():
+    gs = gridspec.GridSpec(2, 1)
+
+    gs.update(left=.1)
+    assert gs.left == .1
+
+    with pytest.raises(AttributeError):
+        gs.update(lleft=.1)
```

Comment:
```suggestion
```
Let's not check this. This is basically an invalid parameter name, and we don't check functions for invalid parameter names. It's slightly different in that the parameters are not explicit, but instead of being tempted to add tests because of that, let's rather make the parameters explicit (see #29956).